src/IssueViz/Models/AnalysisIssueVisualization.cs

```diff
@@ -40,5 +41,9 @@ public AnalysisIssueVisualization(IReadOnlyList<IAnalysisIssueFlowVisualization>
 
         public IReadOnlyList<IAnalysisIssueFlowVisualization> Flows { get; }
         public IAnalysisIssue Issue { get; }
+        public int StepNumber => 0;
+        public bool IsNavigable { get; set; }
+        public IAnalysisIssueLocation Location => Issue;
+        public SnapshotSpan? Span { get; set; }
```

Comment:
The issue viz will also need a span and the location properties for tagging, so I've made it implement the location viz interface.